src/revivalpmmp/pureentities/data/MobTypeMaps.php

```diff
@@ -67,4 +72,4 @@ class MobTypeMaps{
 		"wither_skeleton",
 		"zombie_pigman"
 	);
-}
\ No newline at end of file
+}
```

Comment:
no ending lines

Reply:
I'm editing online in the code editor, for the life of me I can't remove these (nor did I add them)?

Reply:
Confirmed, no empty new line at end of file?